### ISSUE TITLE
"import" report error messages

### ISSUE THREAD

**Issue by crossz** [spyder-ide/spyder#3552]
## Description

**What steps will reproduce the problem?**

1. in the tab of "IPython console"
2. input: "import numpy" or "import scipy.io" etc

**What is the expected output? What do you see instead?**
import successfully with nothing printed or popped.
What I see is the internal console for issue reporting returned as follows:

```
Spyder Internal Console

This console is used to report application
internal errors and to inspect Spyder
internals with the following commands:
  spy.app, spy.window, dir(spy)

Please don't use it to run your code

>>> WARNING:traitlets:kernel died: 3.00099992752
Traceback (most recent call last):
  File "D:\DEV\Anaconda2\lib\site-packages\qtconsole\base_frontend_mixin.py", line 163, in _dispatch
    handler(msg)
  File "D:\DEV\Anaconda2\lib\site-packages\spyder\widgets\ipythonconsole\namespacebrowser.py", line 188, in _handle_execute_reply
    self.handle_exec_method(msg)
  File "D:\DEV\Anaconda2\lib\site-packages\spyder\widgets\ipythonconsole\shell.py", line 225, in handle_exec_method
    properties = ast.literal_eval(data['text/plain'])
  File "D:\DEV\Anaconda2\lib\ast.py", line 49, in literal_eval
    node_or_string = parse(node_or_string, mode='eval')
  File "D:\DEV\Anaconda2\lib\ast.py", line 37, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 28
    'scipy': {'array_ndim': <function numpy.core.fromnumeric.ndim>,
                            ^
SyntaxError: invalid syntax
```

**Please provide any additional information below**

## Version and main components

* Spyder Version: 3.0.0 
* Python Version: 2.7.12
* Qt Versions: 5.6.0, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.5.0 :  1.3.0 (OK)
pep8 >=0.6       :  1.7.0 (OK)
pygments >=2.0   :  2.1.3 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
pandas >=0.13.1  :  0.18.1 (OK)
numpy >=1.7      :  1.11.1 (OK)
sphinx >=0.6.6   :  1.4.6 (OK)
rope >=0.9.4     :  0.9.4 (OK)
jedi >=0.8.1     :  0.9.0 (OK)
matplotlib >=1.0 :  1.5.3 (OK)
sympy >=0.7.3    :  1.0 (OK)
pylint >=0.25    :  1.5.4 (OK)

```

**Comment by ccordoba12** [comment]
Thanks for reporting. This is a duplicate of issue #3528, which is already fixed.

**Comment by ccordoba12** [comment]
And the fix will be part of 3.0.1 :-)
